CITATION.cff

```diff
@@ -0,0 +1,35 @@
+@software{the_jwst_developers_2022_6984366,
+  author       = {Bushouse, Howard and
+                  Eisenhamer, Jonathan and
+                  Dencheva, Nadia and
+                  Davies, James and
+                  Greenfield, Perry and
+                  Morrison, Jane and
+                  Hodge, Phil and
+                  Simon, Bernie and
+                  Grumm, David and
+                  Droettboom, Michael and
+                  Slavich, Edward and
+                  Sosey, Megan and
+                  Pauly, Tyler and
+                  Miller, Todd and
+                  Jedrzejewski, Robert and
+                  Hack, Warren and
+                  Davis, David and
+                  Crawford, Steven and
+                  Law, David and
+                  Gordon, Karl and
+                  Regan, Michael and
+                  Cara, Mihai and
+                  MacDonald, Ken and
+                  Bradley, Larry and
+                  Shanahan, Clare and
+                  Jamieson, William},
+  title        = {JWST Calibration Pipeline1.6.2,
```

Comment:
should there a whitespace before the version number?

Reply:
```suggestion
  title        = {JWST Calibration Pipeline 1.6.2,
```